docs/streams/developer-guide/dsl-api.html

```diff
@@ -2906,6 +2906,7 @@ <h2><a class="toc-backref" href="#id7">Overview</a><a class="headerlink" href="#
                     <tr class="row-even"><td><p class="first"><strong>Transform (values only)</strong></p>
                         <ul class="last simple">
                             <li>KStream -&gt; KStream</li>
+                            <li>KTable -&gt; KTable</li>
```

Comment:
Not sure I follow exactly what doc changes you're after...

I can switch this paragraph to use `ValueTransformerWithKey` rather than `ValueTransformer` and call out that the key is readOnly. Is that what you're after?

Reply:
OK, great.